addons/mail/static/src/discuss/core/common/suggestion_service_patch.js

```diff
@@ -12,6 +12,43 @@ const suggestionServicePatch = {
         const res = super.getSupportedDelimiters(thread);
         return thread?.model === "discuss.channel" ? [...res, ["/", 0]] : res;
     },
+    /**
+     * @override
+     */
+    isSuggestionValid(persona, thread) {
+        if (thread?.model === "discuss.channel") {
+            return persona.type === "partner";
```

Comment:
This is repeating the condition in the super call, so not ideal.

The best would be to specify which condition should apply in this case specifically, so allow odoobot in channel. And let the super call handle the rest.